### ISSUE TITLE
Visually impaired users should have access to information-rich "story" of any garden

### ISSUE THREAD

**Issue by kellydinneen** [kellydinneen/GitHub-Garden#73]
### Description
- users with screenreaders should encounter detailed description of garden upon loading profile

* quantitative: total numbers of flowers, ratios of colors, typical root density and stem height, etc
* stems sway in breeze
* name of GitHub user

- should also encounter vivid description of each flower when tabbing through garden
